etc/cowrie.cfg.dist

```diff
@@ -191,7 +191,16 @@ auth_class = UserDB
 # Source Port to report in logs (useful if you use iptables to forward ports to Cowrie)
 #reported_ssh_port = 22
 
+# ============================================================================
+# Proxy Options
+# ============================================================================
+[proxy]
```

Comment:
to prevent confusion between ssh and telnet, should this be marked somehow as being the ssh proxy ?

Reply:
When adding Telnet I would simply add a _telnet_port_ option and share most of the configs between SSH and Telnet proxies (in particular the pool would be shared between them anyway). But if you prefer them to be completely separate I can do it.

Reply:
Maybe better to have `ssh_port` and `telnet_port`? So there's no implicit `port`. 
Maybe host also needs `telnet_host` and `ssh_host` to have a different one for each?